games-util/pyfa/pyfa-2.9.4.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python3_6 )
```

Comment:
What's blocking 3.7? Maybe I could help.

Reply:
Upstream says it needs 3.6, but I don't know if it would run with 3.7. I tried to test it, but there are at least some dependencies (like dev-python/logbook for example) that don't have 3.7 support in the ebuild currently.